client/src/components/Group.js

```diff
@@ -0,0 +1,73 @@
+import styled from "styled-components";
+import { bool } from "prop-types";
+import { NavLink } from "react-router-dom";
+
+import IconGroup from "./IconGroup";
+import Button from "./Button";
+import IconUser from "./IconUser";
+
+export default function Group({ openGroup, setOpenGroup }) {
+  return (
+    <NavContainer openGroup={openGroup}>
+      <IconGroup />
+      <h4> Groupname </h4>
+      <br />
+      <p> Berlin</p>
+      <p>
+        Apples ducks straw, quail a ostriches donkey, hay hook cucumbers. Lamb
+        pig rooster sheep. . Onion organic orange
+      </p>
+      <Flexbox>
+        <IconUser />
+        <IconUser />
+        <IconUser />
+        <IconUser />
+        <IconUser />
+      </Flexbox>
+
+      <Button>
+        <Link to="./editGroup" onClick={() => setOpenGroup(!openGroup)}>
+          edit Group
+        </Link>
+      </Button>
+      <Button onClick={() => setOpenGroup(!openGroup)}>close Group</Button>
+    </NavContainer>
+  );
+}
+
+const Flexbox = styled.div`
+  display: flex;
+  flex-flow: row wrap;
+  gap: 1vw;
+  padding: 0.5rem;
+`;
+
+const NavContainer = styled.div`
+  padding: 3rem 2rem;
+  position: absolute;
+  background: var(--onetransparent);
+  top: 0;
+  left: 0;
+  right: 0;
+  width: 100vw;
```

Comment:
I'm not sure but is it maybe obsolete. Doesn't `left: 0` and `right: 0`have the same effect as 100vw as it puts the content to 0 on both left and right? 🤔

Reply:
yeah, you're right